modules/develop/pages/data-transforms/deploy.adoc

```diff
@@ -48,6 +47,43 @@ You can deploy your transform function using the xref:reference:rpk/rpk-transfor
 rpk transform deploy
 ----
 
+When the transform function reaches Redpanda, it starts processing new records that are written to the input topic.
+
+[[reprocess]]
+=== Reprocess records
+
+In some cases, you may need to reprocess records from an input topic that already contains data. Processing existing records can be useful if, for example, the output topic was accidentally deleted, and you need to recreate the lost data by reprocessing those existing records.
+
+You can specify the exact point from which the transform function should start processing records in the input topic, starting from a specific offset or a specific timestamp. These options ensure that you can handle various scenarios where reprocessing is necessary to maintain data integrity and resilience.
```

Comment:
>specific offset

There is no option to specify an absolute offset. Records within a given _partition_ are totally ordered, but there's no absolute index for a record within a topic with multiple partitions. So `--from-offset` always specifies an offset delta. The most common usage is probably something like `--from-offset +0`, which means "start from the beginning of each partition".

Lmk if the help text in my other comment is unclear; happy to flesh it out a bit more.

Reply:
>specific timestamp

nitpick (pedantic): first record (per partition of the input topic) committed after the given timestamp. this one is definitely a nitpick.